index.bs

```diff
@@ -288,13 +288,15 @@ spec: webidl; urlPrefix: https://heycam.github.io/webidl/#
 
       <p>
         The <dfn for='VideoConfiguration' dict-member>bitrate</dfn> member
-        represents the average bitrate of the video track given in units of bits
-        per second. In the case of a video stream encoded at a constant bit rate
-        (CBR) this value should be accurate over a short term window. For the
-        case of variable bit rate (VBR) encoding, this value should be usable to
-        allocate any necessary buffering and throughput capability to
-        provide for the un-interrupted decoding of the video stream over the
-        long-term based on the indicated {{VideoConfiguration/contentType}}.
+	represents the bitrate of the video track given in units of bits per second.
+	In the case of a video stream encoded at a constant bit rate (CBR)
+	this shall represent the average bitrate of the video track.
+	For the case of variable bit rate (VBR) encoding, this value shall
+	represent the maximum bitrate of the stream. In either case, this
+	value should be usable to allocate any necessary buffering and
```

Comment:
I find this language somewhat confusing.  The goal of this API is to determine the performance when the browser is given a stream with certain parameters.  Invoking the API does not allocate any codecs or buffers in the browser.  That would be done through other APIs (WebCodecs, MSE, etc.)